plugin/assets/css/src/components/material.css

```diff
@@ -29,6 +29,11 @@
 	direction: ltr;
 	font-feature-settings: "liga";
 	-webkit-font-smoothing: antialiased;
+
+	@nest p & {
```

Comment:
is there a fall back for where nesting is not supported?

Reply:
PostCSS turns this into `p .material-icons` in the compiled file. The final version will never see the `@nest` keyword